src/modeles/descriptionService.js

```diff
@@ -34,6 +34,18 @@ class DescriptionService extends InformationsHomologation {
     this.referentiel = referentiel;
   }
 
+  descriptionFonctionnalites() {
+    return this.fonctionnalites
+      .filter((fonctionnalite) => this.referentiel.fonctionnalites()[fonctionnalite])
+      .map((fonctionnalite) => (
+        this.referentiel.fonctionnalites()[fonctionnalite].description
+      ));
+  }
```

Comment:
Peut-on donner plus de responsabilités au `referentiel` ici ?
Que ce soit lui qui puissent retourner directement le contenu des fonctionnalités ?

Au niveau appelant, ça serait `this.referentiel.fonctionnalites(this.fonctionnalites)`. Et si c'est aussi simple que ça, peut-être déplacer ce _one-liner_ directement dans l'objet vue ?

Reply:
Mise à jour : 
- j'ai ajouté referentiel.descriptionsFonctionnalites(fonctionnalites) au référentiel
- j'ai appelé directement dans vueAnnexePDFDescription